tracer/tracer.go

```diff
@@ -503,126 +575,170 @@ func loadAllMaps(coll *cebpf.CollectionSpec, ebpfMaps map[string]*cebpf.Map,
 	return nil
 }
 
-// loadUnwinders just satisfies the proof of concept and loads all eBPF programs
-func loadUnwinders(coll *cebpf.CollectionSpec, ebpfProgs map[string]*cebpf.Program,
-	tailcallMap *cebpf.Map, includeTracers types.IncludedTracers,
+// loadPerfUnwinders just satisfies the proof of concept and loads all eBPF programs
```

Comment:
Not sure what "satisfies the proof of concept" means, probably rewrite

Reply:
:see_no_evil: sorry for this left over from a rewrite.